vocab/credentials/v2/vocabulary.drawio

```diff
@@ -0,0 +1,501 @@
+<mxfile host="Electron" modified="2023-09-30T08:19:13.594Z" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/22.0.0 Chrome/114.0.5735.289 Electron/25.8.3 Safari/537.36" etag="P2DNnCw8gU-rS8J6qqE1" version="22.0.0" type="device">
+  <diagram name="Page-1" id="5wcF2D67hh1iBqyEtvuE">
+    <mxGraphModel dx="3310" dy="2042" grid="1" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="1" pageScale="1" pageWidth="1600" pageHeight="900" math="0" shadow="0">
+      <root>
+        <mxCell id="0" />
+        <mxCell id="1" parent="0" />
+        <UserObject label="&lt;i&gt;&lt;font color=&quot;#000000&quot;&gt;verifiableCredential&lt;/font&gt;&lt;/i&gt;" link="https://w3.org/2018/credentials/#verifiableCredential" id="1oUjKqBKF74gJ-cnWfdO-3">
```

Comment:
All the links in here are w3.org vs www.w3.org and with a trailing `/`.  The server redirs to www and adds the `/` no matter what you do, but the spec and context say vocab is `https://www.w3.org/2018/credentials#` (https://www.w3.org/TR/vc-data-model-2.0/#vocabularies).  Should these links match the vocab URLs?

Reply:
I have hit some problems with dereferencing these links properly. For some reason, the dereferencing from the embedded version did not work properly without the extra '/'.

@davidlehn, is it o.k. if we raise this as a separate editorial issue marked as post-CR? I would not want to keep this PR on hold (this PR is labeled as before CR)

Cc @msporny

Reply:
Scrap my previous remark. I have solved the issue (in https://github.com/w3c/vc-data-model/pull/1296/commits/b4031ef5d9c8fab6084b098730951992477bdb62), and refreshed the files accordingly. Resolving this conversation as well.

Thanks for these, @davidlehn !